src/Microsoft.AspNetCore.SignalR.Client/HttpConnectionOptions.cs

```diff
@@ -17,8 +18,13 @@ public class HttpConnectionOptions
         internal IDictionary<string, string> _headers;
         internal CookieContainer _cookies;
 
+        public HttpConnectionOptions()
+        {
+            Transport = HttpTransports.All;
+        }
+
         public Uri Url { get; set; }
-        public HttpTransportType? Transport { get; set; }
+        public HttpTransportType Transport { get; set; }
```

Comment:
Should this be pluralized?

Reply:
Yeah, we should be consistent with the server. I like `Transports` here.